### ISSUE TITLE
cookies?

### ISSUE THREAD

**Issue by manageC0d** [IOsonoTAN/simple-facebook-graph-api#2]
token & like cookie.txt Can it be recorded?

**Comment by IOsonoTAN** [comment]
Yes you can, But you have to call to graph api, After that you'll get data from Facebook and then you can store that data into your file.

Thanks

**Comment by manageC0d** [comment]
can you do it? I could not Thanks bro...

**Comment by krissada-paysbuy** [comment]
Hello @batuhanates0,

Yes i can do, Actually i used this lib (in version 2.6) for get all comments, likes and posts on Facebook page and store that data into database. Currently it's work fine and there no problem.

Thanks

**Comment by IOsonoTAN** [comment]
Hello @batuhanates0,

Yes i can do, Actually i used this lib (in version 2.6) for get all comments, likes and posts on Facebook page and store that data into database. Currently it's work fine and there no problem.

Thanks

**Comment by manageC0d** [comment]
okay I am waiting ! thank you..

**Comment by manageC0d** [comment]
You did not do?

**Comment by batuhanates0** [comment]
no cookie :(

add code,

$cookie_file_path = 'cookie.txt';
$cookie_file_path = realpath($cookie_file_path);
curl_setopt($curl, CURLOPT_COOKIESESSION, true);  
curl_setopt($curl, CURLOPT_COOKIEFILE, $cookie_file_path);
curl_setopt($curl, CURLOPT_COOKIEJAR, $cookie_file_path);
